src/components/HistoryPage/HistoryEntry.js

```diff
@@ -22,14 +26,14 @@ export default function HistoryEntry({
               width={'25'}
               height={'25'}
               viewBox="4 4 25 25"
-              onClick={() => setOpenConfirm(true)}
+              onClick={() => toggleConfirm(true)}
```

Comment:
Maybe something like that? If you explicitly set it to true, you might run into bugs at some point. It's nicer to say, that it should be the opposite of what it is right now. In this case `!openConfirm` - I guess.
```suggestion
              onClick={() => toggleConfirm(!openConfirm)}
```

Reply:
Good point!